DuckDuckGo/SiteRatingPrivacyProtectionExtension.swift

```diff
@@ -61,11 +61,19 @@ extension SiteRating {
     }
 
     func majorNetworksBlockedText() -> String {
-        return String(format: UserText.privacyProtectionMajorTrackersBlocked, uniqueMajorTrackerNetworksBlocked)
+        if 1 == uniqueMajorTrackerNetworksBlocked {
+            return UserText.privacyProtectionMajorTrackerBlocked
+        } else {
+            return String(format: UserText.privacyProtectionMajorTrackersBlocked, uniqueMajorTrackerNetworksBlocked)
+        }
     }
 
     func majorNetworksDetectedText() -> String {
-        return String(format: UserText.privacyProtectionMajorTrackersFound, uniqueMajorTrackerNetworksDetected)
+        if 1 == uniqueMajorTrackerNetworksDetected {
```

Comment:
Not a biggie but for readability reasons I'd suggest switching these:
`if uniqueMajorTrackerNetworksDetected equals 1`
reads better than
`if 1 equals uniqueMajorTrackerNetworksDetected`

In the first example the variable provides context. In the second, the reader must wait longer before getting to salient information they need to assess the statement.

Reply:
Agree.  It's an old habit from my first job I've never been able to shake "always test against the constant".